packages/create-block/lib/templates.js

```diff
@@ -24,62 +24,56 @@ const templates = {
 			license,
 			version,
 		},
-		outputFiles: {
-			'.editorconfig': 'editorconfig',
-			'editor.css': 'editor-css',
-			'index.js': 'es5/index-js',
-			'$slug.php': 'es5/plugin-php',
-			'style.css': 'style-css',
-		},
+		outputFiles: [ '.editorconfig', 'editor.css', 'index.js', '$slug.php', 'style.css' ],
 	},
 	esnext: {
-		defaultAnswers: {
+		defaultValues: {
 			namespace,
 			slug: 'esnext-example',
 			title: 'ESNext Example',
-			description: 'Example block written with ESNext standard and JSX support – build step required.',
+			description:
+				'Example block written with ESNext standard and JSX support – build step required.',
 			dashicon,
 			category,
 			author,
 			license,
 			version,
 		},
-		outputFiles: {
-			'.editorconfig': 'editorconfig',
-			'.gitignore': 'gitignore',
-			'editor.css': 'editor-css',
-			'src/index.js': 'esnext/index-js',
-			'$slug.php': 'esnext/plugin-php',
-			'style.css': 'style-css',
-		},
+		outputFiles: [
+			'.editorconfig',
+			'.gitignore',
+			'editor.css',
+			'src/index.js',
+			'$slug.php',
+			'style.css',
+		],
 		wpScriptsEnabled: true,
 	},
 };
 
 const getTemplate = ( templateName ) => {
 	if ( ! templates[ templateName ] ) {
 		throw new CliError(
-			'Invalid template type name.' +
-			` Allowed values: ${ Object.keys( templates ).join( ', ' ) }.`
+			'Invalid template type name.' + ` Allowed values: ${ Object.keys( templates ).join( ', ' ) }.`
```

Comment:
Could be simplified?

```suggestion
			`Invalid template type name. Allowed values: ${ Object.keys( templates ).join( ', ' ) }.`
```

Reply:
Sure, Prettier run will create more cases like this.